src/neptune_scale/api/attribute.py

```diff
@@ -0,0 +1,243 @@
+import functools
+import itertools
+import warnings
+from datetime import datetime
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Collection,
+    Dict,
+    Iterator,
+    List,
+    Optional,
+    Set,
+    Tuple,
+    Union,
+    cast,
+)
+
+if TYPE_CHECKING:
+    from neptune_scale.api.run import Run
+
+
+def warn_unsupported_params(fn: Callable) -> Callable:
+    # Perform some simple heuristics to detect if a method is called with parameters
+    # that are not supported by Scale
+    warn = functools.partial(warnings.warn, stacklevel=3)
+
+    def wrapper(*args, **kwargs):  # type: ignore
```

Comment:
```suggestion
@functools.wraps(fn)
def wrapper(*args, **kwargs): # type: ignore
```

Reply:
I think I already added this in another branch 👍